src/components/Sidebar/RightSidebar/right-sidebar-props.ts

```diff
@@ -0,0 +1,6 @@
+import { MenuData } from './menu-data';
```

Comment:
Only used by RightSidebar, so can just be inline in the `RightSidebar` component